README.md

```diff
@@ -1,6 +1,53 @@
 # Julia925
 Learn Julia in one workday!
 
-Either clone the repo and work local in the notebooks or click on the badge below to open a Binder session.
+If you have *Julia* and *IJulia*-notebooks installed, clone the repo and work local in the notebooks. Otherwise click on the badge below to open a Binder session or check the *installation instructions* below to install *Julia* and *IJulia*-notebooks.
```

Comment:
One sentence on how to install IJulia, e.g.

To install IJulia (or any other Julia package), open the Julia REPL, type `]` (this opens the package manager) and enter
```
add IJulia
```